about.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://kit.fontawesome.com/adf6e8ca6b.js" crossorigin="anonymous"></script>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" href="images/title-ai.png">
+  <title>About</title>
+</head>
+<body>
+  <header>
+    <nav class="nav1">
+              <a class="nav-link" href="#">Logout</a>
+              <a class="nav-link" href="#">My Page</a>
+              <a class="nav-link" href="#">English</a>
+              <a class="nav-link" href="#"><i id="twitter-icon" class="fa fa-twitter text-white"></i></a>
+              <a class="nav-link" href="#"><i id="facebook-icon" class="fa fa-facebook text-white"></i></a>
+      </nav>
+    </nav>
+    <nav class="nav2">
+      <div class="nav2Content">
+        <a id="bordered-link" class="nav-item" href="#">AI Compaign</a>
+        <a class="nav-item" href="#">News</a>
+        <a class="nav-item" href="#">Sponser</a>
+        <a class="nav-item" href="#">Join</a>
+        <a class="nav-item" href="#">Program</a>
+        <a class="nav-item" href="#">About</a>
+      </div>
+      <div class="navLogo"><a href="index.html"><img src="images/Logo.png" alt="logo"></a></div>
+    </nav>
+    <button class="mobile-menu" type="menu" ><img src="https://img.icons8.com/ios/50/000000/menu--v4.png" alt="menu"/></button>
+      <span class="navbar-toggler-close"></span>
+    </button>
+    <div class="mobile-menu-list" id="about-menu">
+      <a class="nav-item" href="#">About</a>
+      <a class="nav-item" href="#">Program</a>
+      <a class="nav-item" href="#">Join</a>
+      <a class="nav-item" href="#">Sponser</a>
+      <a class="nav-item" href="#">News</a>
+      <a class="nav-item" href="#">AI Compaign</a>
+    </div>
```

Comment:
- [ ] Your mobile menu works as expected but there is no way to navigate to the home page from the about page in the mobile version since a home link is absent. It would be great if you could rectify this issue as it should be possible to navigate between the pages.
- Your design:
![C5](https://user-images.githubusercontent.com/85113567/153270814-4c9c8e4f-503d-4ec1-872a-b0cda5e9e61c.PNG)